.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
The most important considerations from the code review are:

1. **Security implications**: Adding `__pycache__/;` to `.gitignore` may inadvertently hide sensitive information, potentially compromising security if an attacker gains access to the repository.

2. **Alternative solutions**: Consider using environment variables or tools like `virtualenv`/`conda` to manage Python cache directories more securely and robustly, rather than relying solely on `.gitignore`.